src/itf/decodable-itf.h

```diff
@@ -86,6 +86,10 @@ class DecodableInterface {
   /// The "frame" starts from zero.  You should verify that IsLastFrame(frame-1)
   /// returns false before calling this.
   virtual BaseFloat LogLikelihood(int32 frame, int32 index) = 0;
+  
+ 
+  //Computes loglikelihoods betewen [0,count) and writes them to out for the frame specified
+  virtual void ComputeLogLikelihoods(BaseFloat* out, int32 frame, int32 count) {};
```

Comment:
I do like this method, because having a contiguous set of features would be useful in many ways, but having a default implementation that does nothing in unacceptable.

We should have the default implementation default to something like:

```
for(int32 i = 0; i < count; ++i) {
  out[i] = this->LogLikelihood(frame, i);
}
```

This way we can extend the interface without breaking any downstream users of Kaldi who have extended this interface themselves. Secondly, if the overhead of all those virtual function calls is a problem for someone, they can still override this.

Reply:
@galv This itf is to be decided. We might remove it after Hang finalizes nnet3-compute. I leave it here temporally.